lib/Mojo/Asset/File.pm

```diff
@@ -116,7 +116,12 @@ sub new {
   return $file;
 }
 
-sub size { -s shift->handle }
+sub size {
+  my $file = shift;
```

Comment:
This should be $self and not $file imo. (same multiple places)

Reply:
Agreed.